lib/Backend/Lower.cpp

```diff
@@ -19473,21 +19472,12 @@ void Lowerer::GenerateFastInlineIsIn(IR::Instr * instr)
         src1Untagged,
         headSegmentLengthOpnd,
         Js::OpCode::BrGe_A,
-        doneFalseLabel,
-        instr);
-
-    InsertCompareBranch(
-        src1Untagged,
-        IR::IntConstOpnd::New(0, src1Untagged->GetType(), this->m_func),
-        Js::OpCode::BrLt_A,
```

Comment:
>BrLt_A [](start = 20, length = 6)

Why is the < check not valid?

Reply:
Because there can be cases like this like in the test I added now
FloatArr0[-7] = 595461181.1;
print(-7 in FloatArr0); // true